x-pack/plugins/observability/public/utils/date.ts

```diff
@@ -9,7 +9,7 @@ export function getAbsoluteTime(range?: string, opts = {}) {
   if (range) {
```

Comment:
Do we still need this if `range` is no longer optional?

Reply:
no, this can be removed. 👍🏻